app/controllers/moab_storage_controller.rb

```diff
@@ -24,14 +16,4 @@ def show
       format.all { render json: @output, content_type: 'application/json' }
     end
   end
-
-  private
-
-  def druids_from_storage_root
-    @druids ||= begin
-      @storage_root ||= "#{Moab::Config.storage_roots}/#{Moab::Config.storage_trunk}"
-      Dir.glob("#{@storage_root}/**/[a-z][a-z]*[0-9]").map { |d| d.split("/").last }
-    end
-  end
-
 end
```

Comment:
This might be better as a separate PR.  I  opened a ticket for "figure out what controller(s) we actually want" since the team isn't yet on the same page there.  

We're removing this code because we don't want to allow such a request against all the production storage_root data, yes?   Maybe comment out for now instead?   Or get rid of the routes?   I agree this code is likely to go away, but it's slightly possible we'll want some form of it.

Reply:
I'm happy to go with this commit and nuke the methods - just FYI. But I agree that it would be better as a separate PR.